README.md

```diff
@@ -50,8 +50,21 @@ Default credentials are set up in `config/config.ini` (`test-user:test-user` by
 
 The PHP [development server][php-web-server] can be run via `php -S localhost:8000`.
 
+### Running with Docker
+
+Install Docker, build the image, then run it.
+
+```
+docker build -t srobo-ide:local .
+docker run -d --rm --network=host -t srobo-ide:local
```

Comment:
As I understand it, using `--rm` here means that the docker image will be deleted when it stops, which means that any logs or content created within the IDE will disappear at that point.

Combined with the `COPY` operation which pulls a current snapshot of the code state into the image, this doesn't feel especially conducive to development to me -- the logs are useful to help diagnose things and I've found that having some repos ("projects" in IDE terms) around is useful so you don't need to start from scratch every time. Certainly I wouldn't want to have to recreate the projects for every code change made!

Reply:
Typically I would mount the host development files in the same place on the container, and then run a shell in the container to take different actions. But I'll document this.